pontoon/base/models.py

```diff
@@ -76,6 +76,10 @@ def cldr_plurals_list(self):
         else:
             return map(int, self.cldr_plurals.split(','))
 
+    @property
+    def dir(self):
```

Comment:
This is a fine way to do this. I like it!

It's interesting though, because if we wanted to be consistent with how we store data for other locales, we'd store this as a new field on the model and add a migration that sets it to true for the 4 locales we care about. But there's no real benefit to that besides being consistent with how we store the rest of the data about a locale.

I honestly can't decided between either choice. What do you think?